### ISSUE TITLE
Difference ruRU localizations

### ISSUE THREAD

**Issue by aztec102** [shagu/pfQuest#136]
Found a difference in the translations of Vanilla (1.12) and Classic (1.13).
As a result, the same NPC is called differently [Kolold Laborer](https://ru.wowhead.com/npc=80/%D0%BA%D0%BE%D0%B1%D0%BE%D0%BB%D1%8C%D0%B4-%D1%80%D1%83%D0%B4%D0%BE%D0%BA%D0%BE%D0%BF) and [Kobold Laborer](https://ru.classic.wowhead.com/npc=80/%D0%BA%D0%BE%D0%B1%D0%BE%D0%BB%D1%8C%D0%B4-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BD%D0%B8%D0%BA)
Because of this, quests, NPCs may not be displayed correctly (example #135 )
The problem is that the addon uses 1.13 localization, although it is not clear why.
The bottom line is that on Russian localization, part of the quest is not shown - as I wrote above.
I think the addon is very cool and few good addons can be found on the old version of the game.

**Comment by aztec102** [comment]
I wrote a script that checks your translation with server translation 1.12
Found a difference in the following positions. I also checked with wowhead.
DB(items)
`Перевод не совпадает ID = 2715, Существо – Предмет, Фонарь – Круглый != "Монстр – Предмет, Фонарь – Круглый", 
Перевод не совпадает ID = 2760, Набор для шитья Турмана != "Набор для шитья Тармена", 
Перевод не совпадает ID = 2830, Останки Турмана != "Останки Тармена", 
Перевод не совпадает ID = 2837, Письмо Турмана != "Письмо Тармена", 
Перевод не совпадает ID = 3455, Короткий меч ловчего смерти != "Короткий меч стража смерти", 
Перевод не совпадает ID = 3515, Посох Атаерика != "Посох Атэрика", 
Перевод не совпадает ID = 4926, Пустой бочонок Чэня != "Пустой бочонок Чена",  
Перевод не совпадает ID = 6446, Сумка из змеиной кожи != "Сумка земеиной кожи", 
Перевод не совпадает ID = 7308, Свиток заклинания проявления != "Свиток заклятия Проявления", 
Перевод не совпадает ID = 13317, Свисток палевого ящера != "Свисток бежевого ящера", 
Перевод не совпадает ID = 16784, Сапфир Аку'мая != "Сапфир Аку'май", 
Перевод не совпадает ID = 17506, Маяк Сквороца != "Маяк Змейера", 
Перевод не совпадает ID = 17507, Маяк Макарча != "Маяк Слидора", 
Перевод не совпадает ID = 22165, Дар любви: Громовой Утес != "Дар обожания: Громовой Утес", 
Перевод не совпадает ID = 22166, Дар любви: Подгорода != "Дар обожания: Подгорода", `

**Comment by shagu** [comment]
Hi,

Thanks for your report.

This addon is using the translations from [VMaNGOS](https://github.com/vmangos/core). VManGOS is the most feature rich, best working and most used public core for 1.12 emulation. If the translations on your server are different, you're either playing on a custom core, a very outdated server revision or your admins don't contribute back to open-source.

>  The problem is that the addon uses 1.13 localization, although it is not clear why.

The real problem is, that there was no 1.12 client in 2006. The ruRU client was a fan project using translations from later expansions and by guessing how it should have been. As there never was an official ruRU 1.12, I assume the 1.13 is the best source we can have.

I have compared all IDs and texts you mentioned from the addon with the VMaNGOS database, and everything looks correct. Both are using the same texts. I don't speak russian, but if you think your translations are the "better" ones, please report them to the VManGOS team to get it fixed there first.

**Comment by aztec102** [comment]
Hi!
Thank you very much for your answer, I thought you won’t answer.
Please share a link to the source file VMaNGOS , I can not find the file from them.

**Comment by shagu** [comment]
Database: 
https://github.com/brotalnia/database

Server:
https://github.com/vmangos/core

Discord:
https://discord.gg/x9a2jt7

**Comment by aztec102** [comment]
Thank you, I will try through them, point out to them the shortcomings of translation

**Comment by aztec102** [comment]
@shagu Maybe use the base https://github.com/WoWruRU/classicdb_ruRU?

**Comment by shagu** [comment]
No, I prefer to have this addon compatible to the majority of servers rather than a less known translation on which I don't know where its even used. If those translations are better than what VMaNGOS ships, I suggest you talking to @ratkosrb (brotalnia) on how to get those into VMaNGOS.

**Comment by ratkosrb** [comment]
The localized texts in the vmangos database were bruteforced from the official Classic 1.13 servers. The only source that can override 1.13 data is original vanilla data, but since there was no official russian client for vanilla, this does not apply here. We are not accepting fan translations for texts that are already bruteforced from official servers.

**Comment by aztec102** [comment]
Thank you very much for unsubscribing, I am very surprised that I got so much attention. I think the question can be closed and I will close it.
